src/components/Staking/StakingConsiderations/index.tsx

```diff
@@ -75,9 +75,18 @@ const StakingConsiderations = ({ page }: StakingConsiderationsProps) => {
     pageData,
     activeIndex,
   } = useStakingConsiderations({ page })
+  
+  const activeStyles = {
+    bg: "var(--eth-colors-offBackground)",
```

Comment:
We should use one of our color tokens here... `offBackground` is on the old list of color tokens we're trying to phase out... Would recommend `background.highlight` here:

`offBackground` = #f7f7f7 in light, #181818 in dark
`background.highlight` = #f7f7f7 in light, #141414 in dark

Also, we don't need to write out the css variable here; we can just use the token name as a string:

```suggestion
    bg: "background.highlight",
```

cc: @nloureiro